.idea/vcs.xml

```diff
@@ -2,7 +2,10 @@
 <project version="4">
   <component name="CommitMessageInspectionProfile">
     <profile version="1.0">
+      <inspection_tool class="BodyLimit" enabled="true" level="WARNING" enabled_by_default="true" />
       <inspection_tool class="GrazieCommit" enabled="true" level="TYPO" enabled_by_default="true" />
+      <inspection_tool class="SubjectBodySeparation" enabled="true" level="WARNING" enabled_by_default="true" />
+      <inspection_tool class="SubjectLimit" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
We can also set 50 characters subject limit by adding `<option name="RIGHT_MARGIN" value="50" />` here, but it is quite restrictive IMO, and GitHub uses 72 characters limit AFAIK